airbyte-integrations/connectors/destination-redshift/src/main/kotlin/io/airbyte/integrations/destination/redshift/typing_deduping/RedshiftDestinationHandler.kt

```diff
@@ -90,12 +90,9 @@ class RedshiftDestinationHandler(
         for (transaction in transactions) {
             val transactionId = UUID.randomUUID()
             if (logStatements) {
-                log.info(
-                    "Executing sql {}-{}: {}",
-                    queryId,
-                    transactionId,
-                    java.lang.String.join("\n", transaction)
-                )
+                log.info {
```

Comment:
these log statements generated a compiler warning, so switch to the non-deprecated version